src/action.ts

```diff
@@ -231,14 +231,38 @@ export const runAction = async (space): Promise<void> => {
     Logger.verbose("No alias changes required");
   }
 
+  // "closed" action happens on PR close and PR merge
+  const githubAction = github.context.payload.action
+
+  // If CDA token is created and we want to purge ephemeral env upon close/merge
+  // then ephemeral token should be deleted as well
+  if ( 
+    DELETE_FEATURE && 
+    CREATE_CDA_TOKEN && 
+    githubAction === "closed" 
+  ) {
+    const { items: keys } = await space.getApiKeys();
+    keys.map((key) => {
+      if (key.name === tokenKeyName) {
+        try {
+          key.delete()
```

Comment:
Does this need to be `await`ed?

Reply:
not sure, but it definitely works like this fine 
can I leave it like it is?

Reply:
So, it executes fine without the await, but if it's an async function (best I can tell, it is), then the catch will never catch the response and you will have no indication that there was an error. Fire & forget vs fire & wait. 

If your intent is to fire & forget, and therefore ignore any failures, then would want to indicate that in comments.

Reply:
I definitely do not want to "fire&forget", but rather catch error/exception if any.
seems like `.then().catch()` contraption is the way to go.

Reply:
just `await`-ing on delete() 👍